pmultiqc/modules/quantms/quantms.py

```
@@ -1838,6 +2015,261 @@ def getSpectrumCountAcrossRep(condition_count_dict: dict):
+            if len(mzid_data) == 0:
+                raise ValueError("Please check your MzIdentML", m)
+            log.info("{}: Done parsing MzIdentML file {}.".format(datetime.now().strftime("%H:%M:%S"), m))
+            m = self.file_prefix(m)
+            log.info("{}: Aggregating MzIdentML file {}...".format(datetime.now().strftime("%H:%M:%S"), m))
+
+            mzid_dict = list()
+            for mzid_tmp in mzid_data:
+                mzid_tmp_part = {k: v for k, v in mzid_tmp.items() if k not in ["SpectrumIdentificationItem"]}
+                for spectrum_item in mzid_tmp.get("SpectrumIdentificationItem", []):
+                    spectrum_item_part = {k: v for k, v in spectrum_item.items() if k not in ["PeptideEvidenceRef", "PeptideSequence"]}
+                    if spectrum_item_part.get("rank", None) == 1 and \
```

Ruff

Use `spectrum_item_part.get("rank")` instead of `spectrum_item_part.get("rank", None)`

Replace `spectrum_item_part.get("rank", None)` with `spectrum_item_part.get("rank")`

(SIM910)